astropy/stats/tests/test_funcs.py

```diff
@@ -465,11 +487,11 @@ def test_poisson_conf_frequentist_confidence_gehrels_2sigma():
                     (8, 2, 3.391, 15.94),
                     (9, 2, 4.046, 17.27),
                     (10, 2, 4.719, 18.58)])
-    assert_allclose(funcs.poisson_conf_interval(nlh[:,0],
-                        sigma=2,
-                        interval='frequentist-confidence').T,
-                    nlh[:,2:],
-                    rtol=0.01)
+    assert_allclose(
+        funcs.poisson_conf_interval(nlh[:, 0], sigma=2,
+                                    interval='frequentist-confidence').T,
```

Comment:
what about leaving ``funcs.poisson...`` in the first line and  adding the new line before ``nlh``? Then ``interval`` can be lined up under it and fit into the 80.